src/components/alert/alert.stories.tsx

```diff
@@ -2,14 +2,17 @@ import React, { useState } from "react";
 import { ComponentStory } from "@storybook/react";
 import Alert from ".";
 import Button from "../button";
+import Box from "../box";
 import isChromatic from "../../../.storybook/isChromatic";
 
 const defaultOpenState = isChromatic();
 
-const AlertComponent: ComponentStory<typeof Alert> = () => {
+// Added to avoid default export warning which causes storybook to not display `show code` examples - https://github.com/storybookjs/storybook/issues/8104
+// eslint-disable-next-line import/prefer-default-export
+export const DefaultAlert: ComponentStory<typeof Alert> = () => {
   const [isOpen, setIsOpen] = useState(defaultOpenState);
   return (
-    <>
+    <Box>
```

Comment:
going to approve this anyway, as it doesn't really matter, but I'm not sure what the benefit is in replacing a fragment with a `Box` with no attributes. This will just put an extra `div` in the markup for no particular reason, won't it?

Reply:
Made the change as per the conversation we had 😄